### PR TITLE
Add mitigation for V-3344

### PR DESCRIPTION
**Pull request by Jack-McDowell**

### CODE REVIEW

**Comment by Jack-McDowell** [comment]
See #179

File: BLUESPAWN-client/src/user/BLUESPAWN.cpp [hunk 1]
```diff
@@ -67,6 +68,7 @@ Bluespawn::Bluespawn() {
 	MitigateV1153* v1153 = new MitigateV1153(mitigationRecord);
 	MitigateV3338* v3338 = new MitigateV3338(mitigationRecord);
 	MitigateV3340* v3340 = new MitigateV3340(mitigationRecord);
+	MitigateV3344* v3344 = new MitigateV3344(mitigationRecord);
```
Comment (ION28) [inline BLUESPAWN-client/src/user/BLUESPAWN.cpp]: Codacy Issue found: [Variable 'v3344' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=4972089)

**Comment by ION28** [comment]
Need to update comment in header file for mitigation to be right class and accurately describe mitigation

**Comment by ION28** [comment]
Need to also change this mitigation to apply for all versions

**Comment by ION28** [comment]
Closing because this branch was pulled into other pr

**Pull request closed without merging**